src/main/steps/applicant1/you-can-use-english-welsh-courts/content.ts

```diff
@@ -75,29 +76,41 @@ const cy = en;
 
 export const form: FormContent = {
   fields: userCase => {
+    // Create map of connections to labels:
+    const connectionsLabelMap = new Map<JurisdictionConnections, Label>();
+    jurisdictionConnectionList.forEach((key: JurisdictionConnections, i: number) =>
+      connectionsLabelMap.set(key, l => l.connectionCheckboxes[i])
```

Comment:
probably better to use 'index' or another name for I so that it doesn't get confused with connection I